src/transformation/CMakeLists.txt

```diff
@@ -17,7 +17,9 @@ target_link_libraries(k4a_transformation PUBLIC
     )
 
 if ("${CMAKE_C_COMPILER_ID}" STREQUAL "GNU" OR "${CMAKE_C_COMPILER_ID}" STREQUAL "Clang")
-    target_compile_options(k4a_transformation PRIVATE "-msse4.1")
+    if ("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "amd64.*|x86_64.*|AMD64.*|i686.*|i386.*|x86.*|amd64.*|AMD64.*")
```

Comment:
Does case matter here? `amd64.*` is listed twice, so is `AMD64.*`

Reply:
At a minimum here, it does look like `amd64` and `AMD64` are duplicated.

Reply:
Case does matter here but good call on the `amd64`, fixed that.